packages/components/ui/src/drivers/fonts/stores.js

```diff
@@ -0,0 +1,54 @@
+import * as _ from 'lamb';
+import {derived, get, writable} from 'svelte/store';
+
+import {
+	getFamilyNames,
+	isFirstToLoad,
+	getRuntimeFamilies
+} from './utils';
+
+// main stores
+export const _firstFamilyToLoad = writable('');
+export const _fontFamilies = writable({});
+export const _status = writable({
+	isFirstLoaded: false,
+	isDone: false
+})
+
+// deriveds
+export const _fontFamilyNames = derived(
+	_fontFamilies,
+	getFamilyNames
+);
+export const _runtimeFamilies = derived(
+	[_fontFamilies, _firstFamilyToLoad],
+	([fontFamilies, firstFamilyToLoad]) => _.pipe([
+		_.collect([
+			_.filterWith(isFirstToLoad(firstFamilyToLoad)),
+			_.filterWith(_.not(isFirstToLoad(firstFamilyToLoad)))
+		]),
+		_.flatten,
+		getRuntimeFamilies
+	])(fontFamilies));
```

Comment:
I think this:

```js
_.collect([
	_.filterWith(isFirstToLoad(firstFamilyToLoad)),
	_.filterWith(_.not(isFirstToLoad(firstFamilyToLoad)))
]),
_.flatten,
```

could be reduced to 

```js
_.sortWith([_.sorterDesc(isFirstToLoad(firstFamilyToLoad))])
```

But I'd prefer something like this:
- note that arguments are sorted alphabetically
- I'm using `isFamilyEqualTo` as suggested elsewhere in the PR

```suggestion
export const _runtimeFamilies = derived(
	[_firstFamilyToLoad, _fontFamilies],
	([firstFamilyToLoad, fontFamilies]) => {
		const isFirstFamily = isFamilyEqualTo(firstFamilyToLoad);

		const runtimeFamilies = _.reduce(
			fontFamilies,
			(acc, {family, faces}) => {
				const fontFaces = faces.map(
					({src, descriptors}) => new FontFace(family, src, descriptors)
				);
				const isFirstFontFace = isFirstFamily(family);

				if (isFirstFontFace) {
					acc.unshift(fontFaces)
				} else {
					acc.push(fontFaces)
				}

				return acc;
			},
			[]
		);

		return runtimeFamilies;
	}
);
```